clang-tools-extra/test/clang-tidy/infrastructure/header-filter-from-config-file/simple/.clang-tidy

```diff
@@ -0,0 +1 @@
+HeaderFilterRegex: '.*'
```

Comment:
It appears this use case was broken by this [patch](https://github.com/llvm/llvm-project/commit/0249554ee1ac49e6f1d93fa78a55971fc706f635),  @HerrCai0907  . 

There may be more things to fix.